bin/geonames-sql.js

```diff
@@ -1,9 +1,8 @@
 #!/usr/bin/env babel-node
 import fs from 'fs';
-
```

Comment:
this newline was to delimit core-modules from npm-modules. (no need to revert this for pull-request though. this is minor)